README.md

```diff
@@ -42,6 +34,22 @@ $ sudo apt update
 $ sudo apt install conjure-up
 ```
 
+## alternative installation
+We use snap's for packaging and installation, as of snapd 2.11 the following
```

Comment:
how about something like "If you want to try the snap distribution, you can install it with 'sudo snap install conjure-up --devmode', and if you want to use it for the openstack-nclxd spell to run openstack in LXD containers on localhost, you also need to run the following three commands: